apps/HowtoAddNewApp.md

```diff
@@ -0,0 +1,185 @@
+# How to add new application
+
+Five files, **source file**, [**Kconfig**](#kconfig), [**Kconfig_ENTRY**](#kconfig_entry), [**Make.defs**](#makedefs) and [**Makefile**](#makefile) are mandatory to add a new application.
+
+## Kconfig
+This file contains configurations of new application to enable or to set specific value.  
```

Comment:
This file is required by Kconfig-frontends menu build system. It is used to specify various config values according to which the application should be built. The following two config values must be defined in the Kconfig file.